src/components/home/SearchInput.tsx

```diff
@@ -58,6 +64,13 @@ const SearchInputComponent: React.FunctionComponent = () => {
     setSearchQueryInputValue(value);
   }
 
+  function onTagClick(event: React.SyntheticEvent) {
+    setSearchQueryInputValue(event.currentTarget.textContent);
+  }
+
+  const tags = suggestedTags.map(tag =>
+    <a href="#" className="badge badge-signal" onClick={onTagClick}>{tag}</a>
```

Comment:
We actually have a `<Tag>` component you could consider using here.

Reply:
Yes, but the styling was too heavy for use in the home page, and with the addition of a link, I didn't think it was worth using it.